### ISSUE TITLE
Out-of-the-Box GEOS-Chem Classic TOMAS simulations fail due to species not found in restart file

### ISSUE THREAD

**Issue by yantosca** [geoschem/geos-chem#2250]
### Name and Institution (Required)

Name: Bob Yantosca
Institution: Harvard + GCST

### Confirm you have reviewed the following documentation

- [x] [Support guidelines](https://geos-chem.readthedocs.io/en/stable/help-and-reference/SUPPORT.html)
- [x] [User manuals](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/related-docs.html)
- [x] [Debugging GEOS-Chem and HEMCO errors](https://geos-chem.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/debug-guide.html)
- [x] [Current and past Github issues](https://github.com/geoschem/geos-chem/issues)

### Description of your issue or question

Out-of-the-box GEOS-Chem Classic TOMAS15 runs fail with this error:

```console
Program received signal SIGSEGV: Segmentation fault - invalid memory reference.

Backtrace for this error:
 
HEMCO ERROR: Cannot find field SPC_BUTDI. Please check variable name in the config. file
 
HEMCO ERROR: Error encountered in routine HCOIO_Read!
 
HEMCO ERROR: Error in HCOIO_DATAREAD called from HEMCO ReadList_Fill: SPC_BUTDI
 --> LOCATION: ReadList_Fill (HCO_ReadList_Mod.F90)
 
HEMCO ERROR: Error in called ReadList_Fill (7) from HEMCO ReadList_Read
 --> LOCATION: ReadList_Read (HCO_ReadList_Mod.F90)
 Error in ReadList_Read called from hco_run
===============================================================================
GEOS-Chem ERROR: Error encountered in "HCO_Run"!
 -> at HCOI_GC_Run (in module GeosCore/hco_interface_gc_mod.F90)

THIS ERROR ORIGINATED IN HEMCO!  Please check the HEMCO log file for 
additional error messages!
===============================================================================
```
This is because the time-cycle setting for the restart file is set to `EFYO` instead of `EY`, and there are some species (e.g. the furans) not in the TOMAS restart file.  We can fix this by adding TOMAS15 to the list of simulations in where we change `EFYO` -> `EY` in `createRunDir.sh`.  I will add this to the PR #2072.

**Comment by yantosca** [comment]
Now fixed in commit dc95553.

**Comment by yantosca** [comment]
We can now close this issue since a fix for this issue has been included in commitdc95553d04b408f6f9bc73f11dc5f0696000349b, which has now been merged (along with PR #2072) into the GEOS-Chem 14.4.0 development stream.
